packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -228,7 +228,9 @@ describeNoCompat("blobs", (getTestObjectProvider) => {
         }
     });
 
-    it("works in detached container", async function() {
+    itExpects("works in detached container", [
+        {"eventName": "fluid:telemetry:Container:ContainerClose",  "error": "0x202"}
```

Comment:
Tests should not expect asserts. Likely the code path should throw a UsageError, rather than abuse assert

Reply:
#9039

Reply:
#9040